### ISSUE TITLE
BUG: Threading issues

### ISSUE THREAD

**Issue by larsoner** [mne-tools/mne-qt-browser#19]
Running
```
import mne
mne.viz.set_browser_backend('pyqtgraph')  # TODO: MNE_BROWSE_ but set_browser (r)
fname = mne.datasets.sample.data_path() + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
raw = mne.io.read_raw_fif(fname)
for _ in range(10):
    raw.plot()
```
I get a bunch of plots that show up, some of which are correct:

![Screenshot from 2021-11-04 13-20-16](https://user-images.githubusercontent.com/2365790/140388245-48f3a770-9030-4467-be3a-0212c9900353.png)

And a bunch of which never finish loading:

![Screenshot from 2021-11-04 13-20-24](https://user-images.githubusercontent.com/2365790/140388263-bf6bd0c4-3df4-4ac6-b149-df77d984d9da.png)

I suspect that this is a threading problem.

**Comment by marsipu** [comment]
Yes, it might be that the signals are somehow confused between the multiple instances of plot. I'll try to find a solution, but I won't manage to do it this week.

A quick solution for the scraper would probably be to set `precompute=False` if that's possible.

**Comment by larsoner** [comment]
We would have to modify all examples to have that parameter I think :(

Better to fix the bug

**Comment by marsipu** [comment]
@larsoner You're right, `MNE_BROWSE_BACKEND` and mne.viz.set_browser_backend are inconsistent.
Personally I prefer `browser` over `browse` because it refers to the object not the activity. WDYT?

Would you update this in 0.24-maint or 1.0-dev?

**Comment by larsoner** [comment]
Can you make a bugfix PR to `mne-tools/mne-python:main` to adjust the config var name? Once that's merged we'll backport to `maint/0.24` the standard way (usually a maintainer just cherry-picks the merged commit to `maint/0.24` and pushes)

**Comment by marsipu** [comment]
> Can you make a bugfix PR to mne-tools/mne-python:main to adjust the config var name? Once that's merged we'll backport to maint/0.24 the standard way (usually a maintainer just cherry-picks the merged commit to maint/0.24 and pushes)

Done
